internal/osutils/osutils.go

```diff
@@ -32,22 +33,28 @@ func CmdString(c *exec.Cmd) string {
 // BashifyPath takes a windows style path and turns it into a bash style path
 // eg. C:\temp becomes /c/temp
 func BashifyPath(absolutePath string) (string, error) {
-	if absolutePath[0:1] == "/" {
+	if len(absolutePath) > 0 && absolutePath[0:1] == "/" {
 		// Already the format we want
 		return absolutePath, nil
 	}
 
-	if absolutePath[1:2] != ":" {
+	if len(absolutePath) < 2 || absolutePath[1:2] != ":" {
 		// Check for windows style paths
 		return "", errs.New("Unrecognized absolute path format: %s", absolutePath)
```

Comment:
Wouldn't this almost certainly happen? Seems we would need to drop this if we want windows compatibility.

Reply:
This was only needed for attempting to bashify any environment variable that looked like a path. Some env vars are empty or were single characters, so I was getting panics without this test. Anyway, I'm not doing this anymore, so I've removed the check.